models/library_item.py

```diff
@@ -11,3 +12,7 @@ class LibraryItem(utils.db.Base):
 
     def __repr__(self):
         return str(self.__dict__)
+
+    def calculate_rent_price_per_day(self) -> float:
+        result = self.pledgePrice / self.quantity
+        return percentage_from(number=result, percent=25)
```

Comment:
It's better to move the percentage value into constant.

Reply:
Resolved